Fix the update-po task

# What needs to be done

**Reviewer:** @patiencedaur 

# Definition of done

- [ ] `make update-po` updates localization files

# Issue

docker run ... `make update-po` results in an error:

```
[ 50%] Built target update-pot
[100%] Regenerate localization files from templates
Traceback (most recent call last):
  File "/usr/local/bin/sphinx-intl", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1656, in invoke
    super().invoke(ctx)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/sphinx_intl/commands.py", line 189, in main
    cfg = read_config(ctx.config, tag)
  File "/usr/local/lib/python3.8/site-packages/sphinx_intl/commands.py", line 45, in read_config
    execfile_(os.path.basename(path), namespace)
  File "/usr/local/lib/python3.8/site-packages/sphinx_intl/pycompat.py", line 57, in execfile_
    exec(code, _globals)
  File "conf.py", line 173, in <module>
    with open('../prolog.rst', 'r') as prolog:
FileNotFoundError: [Errno 2] No such file or directory: '../prolog.rst'
make[3]: *** [CMakeFiles/update-po.dir/build.make:77: CMakeFiles/update-po] Error 1
make[2]: *** [CMakeFiles/Makefile2:320: CMakeFiles/update-po.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:327: CMakeFiles/update-po.dir/rule] Error 2
make: *** [Makefile:215: update-po] Error 2
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -240,15 +240,21 @@ add_custom_target(update-pot
     COMMENT "Update translation sources"
 )
 
-add_custom_target(update-po
+add_custom_target(update-po-raw
     COMMAND "${SPHINX_INTL_EXECUTABLE}"
         update
         -p "${CMAKE_CURRENT_SOURCE_DIR}/locale/en/"
         -d "${CMAKE_CURRENT_SOURCE_DIR}/locale/"
         -l "ru"
-    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/locale
     COMMENT "Regenerate localization files from templates"
     DEPENDS update-pot
+)
+
+add_custom_target(update-po
+    COMMAND make
+        update-po-raw
+        cleanup-po
 )
 
 add_custom_target(cleanup-po

